spec/features/events_spec.rb

```diff
@@ -145,6 +145,19 @@
     end
   end
 
+  scenario "logged in as Organizer I can push the accept all button to accept all applicants" do
+    login(:organizer)
+    @event.max_participants = 3
+    2.times do |i|
+      @pupil = FactoryGirl.create(:profile)
+      @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
```

Comment:
You're creating two applications of the same pupil, aren't you? That could lead to a problem once that isn't possible anymore.